npm-packages/meteor-installer/config.js

```diff
@@ -31,6 +31,15 @@ if (isWindows() && !localAppData) {
   throw new Error('LOCALAPPDATA env var is not set.');
 }
 
+const shouldSetupExecPath = () => {
+  // If the variable isn't set, treat it as true, otherwise coerce it
+  if (!Object.prototype.hasOwnProperty.call(process.env, 'npm_config_meteor_setup_exec_path')) {
```

Comment:
This flag is better as an opt-out of the default behavior, something like:

`--ignore-meteor-setup-exec-path`

So this code here could be simplified to 

`return process.env.npm_ignore_meteor_setup_exec_path`